trees/trees.go

```diff
@@ -119,7 +119,7 @@ func LogHasher(tree *trillian.Tree) (merkle.LogHasher, error) {
 	}
 
 	switch tree.HashStrategy {
-	case trillian.HashStrategy_RFC_6962:
+	case trillian.HashStrategy_RFC6962_SHA256:
 		return rfc6962.New(hash), nil
```

Comment:
This should be `return rfc6962.New(crypto.SHA256), nil`. No need to call `Hash(tree)` above either.